### DISCUSSION TITLE
PBS Post Install Script doesn't kill the nag module

### DISCUSSION THREAD

**Discussion by OddSquirrel** [tteck/Proxmox#900] [Help Needed]
I just ran the PBS Post Install Script on my Proxmox Backup Server 2.3-2 and noticed that it couldn't do away with the subscription nag module. Everything else seems to work just fine.

This isn't an emergency at all, but maybe somebody could check it out whenever possible.

**Comment by tteck** [comment]
Did you clear browser cache?

**Reply by OddSquirrel** [reply]
Hey @tteck, thanks for the quick response.

Yes, that didn't change anything. Also used a backup browser that I never used before to visit my PBS.

**Reply by tteck** [reply]
Ready for you to test again

**Reply by OddSquirrel** [reply]
Still there, unfortunately. 🙁

**Reply by tteck** [reply]
Strange

**Reply by OddSquirrel** [reply]
IKR? As odd as it is, I do understand that you can't drop everything else for it. After all, it's just an annoyance.

**Reply by gyrocoptic** [reply]
Same experience for me.  But I disagree with your assessment that it is not an emergency, I truly don't know if I can live with the nagging, I might just switch to backing up my cluster using paper and pencil. > 01001011101000100101110010010010010010010000010111010110...

**Reply by tteck** [reply]
As I don't use PBS, if I had the `/usr/share/javascript/proxmox-widget-toolkit/proxmoxlib.js` file I could fix any issues.

Edit: It used to be the same as Proxmox VE.

**Comment by ilium007** [comment]
Where is the PBS post install script? I can't find it in the repo anywhere.

**Reply by tteck** [reply]
https://helper-scripts.com/scripts?id=Proxmox+Backup+Server+Post+Install

**Reply by ilium007** [reply]
Thanks - it was right in front of me!
